monitor/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '2'
+services:
+  iotex:
+    container_name: iotex
+    image: iotex/iotex-core:v0.10.0
```

Comment:
it should not hard coded to v0.10.0 but the version user selected when running the script